README.md

```diff
@@ -15,11 +15,25 @@ GoCert requires 3 files to operate:
 * A private key
 * A TLS certificate with that private key
 * A YAML config file with the required parameters
-as an example:
+
+You can generate the cert and the associated key by running:
+`openssl req -newkey rsa:2048 -nodes -keyout key.pem -x509 -days 1 -out cert.pem -subj "/CN=example.com"`
+GoCert does not support insecure http connections.
+
+### Config File
+The config file requires the following parameters:
+* keypath: this is the path to the private key you've generated.
+* certpath: this is the path to a certificate for enabling HTTPS connections.
+* dbpath: the path to a sqlite database file. If the file does not exist GoCert will attempt to create it.
```

Comment:
```suggestion
* dbpath: path to a sqlite database file. If the file does not exist GoCert will attempt to create it. (string)
```

Reply:
Took all of the suggestions and converted this into a table. [Here's](https://github.com/canonical/gocert/blob/TLSENG-239-pebble-notices/README.md) the preview.